.github/workflows/nightly-e2e.yml

```diff
@@ -1,9 +1,7 @@
 # Run integration tests nightly on main
-
-# !! Relevant changes to this file should be propagated manual-integration.yml
-
 name: nightly-e2e-main
 on:
+  workflow_dispatch:
```

Comment:
@MSalopek this should remove the need for manual-e2e.yml, right?

Reply:
Yes